### PR TITLE
[SYNC-98][FE] style: Reading page (Article.vue) styles and feature update

### PR DESCRIPTION
**Pull request by andyrochi**
## Description: 
- Updated various styles pointed out by UI/UX team
## Changes:
- Updated category bar font style and total width (1024px)
- Modified title block layout
- Fixed timeline CSS issue: aligned circles/arrow with border
   - Made timeline column wider 
- Updated tip-tap editor font styles on reading page
- Modified news recommendation styles and filled placeholder with real data
- Updated citation section styles
- Updated all icons on reading page
## Test Scope:
- Any article page
## Screenshots (optional)
- whole page layout
image
- timeline arrows and circles
image
- news recommendation (clickable)
image
- citation section style update
image
- Blockquote update
image

### CODE REVIEW

File: src/assets/scss/main.scss [hunk 1]
```diff
@@ -18,8 +18,22 @@ html, body {
...
-  font-weight: 900;
+h1 {
+  font-weight: 700;
+  font-size: 36px;
+}
+
+h2 {
+  font-weight: 700;
+  font-size: 26px;
+}
```
Comment (newb1er) [inline src/assets/scss/main.scss:29]: UI/UX Team specified letter spacing and color setup.
Should this be set or it's the same as the default value?

```suggestion
h1 {
  font-weight: 700;
  font-size: 36px;
  letter-spacing: 2px;
  color: #0E0E0E;
}

h2 {
  font-weight: 700;
  font-size: 26px;
  letter-spacing: 4px;
  color: #0E0E0E;
}
```

File: src/components/Editor/TiptapEditor.vue [hunk 2]
```diff
@@ -157,6 +157,29 @@ export default {
 
 .editor__content {
   margin-top: 0.5rem;
+  p {
+    font-size: 18px;
+    line-height: 30px;
+  }
```
Comment (newb1er) [inline src/components/Editor/TiptapEditor.vue:163]: UI/UX reported that text in paragraph isn't #000000 (black).
圖片
```suggestion
  p {
    font-size: 18px;
    line-height: 30px;
    color: #000000;
  }
```

File: src/views/Article.vue [hunk 3]
```diff
@@ -360,33 +371,40 @@ p {
...
+  margin-bottom: 16px;
 }
 
-#title-text {
+h1.title-text {
   font-size: 36px;
   font-weight: bold;
   line-height: 56px;
+  margin-bottom: 10px;
 }
```
Comment (newb1er) [inline src/views/Article.vue:385]: Can CSS specificity be reduced?
```suggestion
.title-text {
  font-size: 36px;
  font-weight: bold;
  line-height: 56px;
  margin-bottom: 10px;
}
```

**Review by newb1er — COMMENTED** [review]
Several questions commented. Check it out. 😎

File: src/assets/scss/main.scss (hunk 1 above)
Comment (andyrochi) [reply src/assets/scss/main.scss:29]: Originally, I was thinking about changing all of the colors/styles at once. But sure, thanks for the suggestion!

File: src/components/Editor/TiptapEditor.vue (hunk 2 above)
Comment (andyrochi) [reply src/components/Editor/TiptapEditor.vue:163]: Thanks!

**Comment by andyrochi** [comment]
@newb1er @jinan-tw Updated, please check! Thanks!

**Review by jinan-tw — APPROVED** [review]

**Pull request merged**